packages/vite-node/src/externalize.ts

```diff
@@ -1,10 +1,14 @@
-import { existsSync } from 'node:fs'
-import { isValidNodeImport } from 'mlly'
-import { join } from 'pathe'
+import { existsSync, promises as fsp } from 'node:fs'
+import { readPackageJSON } from 'pkg-types'
```

Comment:
pkg-types uses mlly, so there is really no reason to remove mlly now

we already have an optimized function to ready package.json's: I suggest you port it to vite-node from: packages/vitest/src/runtime/external-executor.ts

Reply:
I didn't realize that `pkg-types` and `mlly` depend on each other. 😅